tests/integration/test_arctic.py

```diff
@@ -144,3 +144,9 @@ def test_default_mongo_retry_timout():
     with pytest.raises(LibraryNotFoundException):
         Arctic('unresolved-host', serverSelectionTimeoutMS=0)['some.lib']
     assert time.time() - now < 1.
+
+def test_stringify_tickstore_library(mongo_host):
```

Comment:
This test should really live on a test file with the sae name as the module under test.  `test_tickstore.py` or similar...